nat-lab/tests/test_pmtu.py

```diff
@@ -14,6 +14,13 @@
 from utils.router import IPStack
 
 
+def features() -> Features:
+    f = test_features(lambda builder: builder.enable_pmtu_discovery())
```

Comment:
Would `enable_xxx` be required if we want to add any feature other than what is there by default ?

Reply:
This pattern was changed to not use lambda+builder like here, but I guess the question still stands. 

Using `pmtu_discovery` as an example: `pmtu_discovery is disabled by default so if you want to enable it, you need to set `enable_pmtu_discovery=True`, which would then enable it with the default value for `response_wait_timeout_s` which is 5. If you wanted a different value for `response_wait_timeout_s`, you could skip `enable_pmtu_discovery=True` and instead do `features.pmtu_discovery=FeaturePmtuDiscovery(<your value here>)` to set it afterwards. But if we instead think about `nurse`, which has many more fields, then it makes more sense to `enable_nurse=True` so you get the default values for all of its fields and then update the fields you want to change, because otherwise you have to create the whole `FeatureNurse` object from scratch